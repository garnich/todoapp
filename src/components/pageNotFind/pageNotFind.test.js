import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import PageNotFind from './pageNotFind'

Enzyme.configure({ adapter: new Adapter() })

describe('Test PageNotFind component', () => {
  it('<PageNotFind>', () => {
    const wrapper = shallow(<PageNotFind />)

    expect(wrapper).toMatchSnapshot()
  })
})
