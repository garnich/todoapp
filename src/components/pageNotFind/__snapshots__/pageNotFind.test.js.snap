// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PageNotFind component <PageNotFind> 1`] = `
<div
  className="col-12 not_find_page"
>
  <p>
    Page not found
  </p>
  <p>
    404
  </p>
  <Link
    to="/app"
  >
    <button
      className="btn btn-outline-secondary btn-lg mt-4 mb-4"
    >
      To Home page
    </button>
  </Link>
</div>
`;
